_posts/2022-02-03-visual-thinking.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: "Visual thinking"
+author: [michal.kowalcze]
+tags: [tech, communication, visualization, planning, creativity]
+---
+
+We use written (source code) language to express our intentions in a machine-readable form. We use spoken language to
+communicate with other people. We pride ourselves as ones choosing programming language optimized to the task at hand.
+Do we use the most optimal way to express our ideas?
+
+## The Story
+
+### Spoken planning
+
+— Josh, could you summarize what we are going to implement?
+
+That was a bit unexpected. He was just trying to match what he already knew from onboarding days with what was just
+said. It was really difficult to follow team discussion at the same time. It was a standard planning session, held over
+Zoom, with his distributed team. Someone was already sharing screen and the story summary, along with acceptance
+criteria, was visible. They were developing an online store and the current topic was: basket price reduction for active
+users. In short: users who spent more than 50€ for the last 7 days should have a discount applied during the checkout
+process.
+
+— Well, we are going to add a new service which is going to hold this discount logic. We will provide an API for the
+cart service to call us and we will check the transactions store for recent orders.<br/>
+— Is that all?<br/>
+— I didn’t catch more changes.<br/>
+— What about customers willing to check if they are eligible for a discount?<br/>
+— Oh, so it seems we need to change ‘my account’ page as well.<br/>
+— And the checkout service? We have to both display discount and use it.<br/>
+— You’re right! I was trying to picture the main change in my mind and wasn’t paying attention to the whole
+discussion.<br/>
+— Guys, checkout stays the same. Cart is providing everything the checkout requires.
+
+### Retrospective
+
+Let’s stop here for a moment. Have you ever been in a situation, when you were forced to do two things at the same time?
+For example listening to what is being said and trying to actively participate in the discussion regarding a not exactly
+well known topic? Was it a demanding experience? I had such an opportunity and I remember these sessions as quite
+demanding. Usually after a hourly session I was exhausted and in need of a break. Not to mention that it required a
```

Comment:
```suggestion
demanding. Usually, after an hourly session, I was exhausted and in need of a break. Not to mention that it required a
```
To jest ciekawe, bo mimo że mamy spółgłoskę to z powodu brzmienia lepiej pasuje tutaj `an`.